vms/platformvm/txs/base_tx_test.go

```diff
@@ -41,13 +41,34 @@ func TestBaseTxMarshalJSON(t *testing.T) {
 		Memo: []byte{1, 2, 3},
 	}}
 
-	txBytes, err := json.Marshal(tx)
+	txJSONBytes, err := json.MarshalIndent(tx, "", "\t")
```

Comment:
baseTx is a component shared with X-chain. I'd move this test to the package where BaseTx is defined.

Reply:
This is testing the serialization of `platformvm.BaseTx` specifically, not `avax.BaseTx`